README.md

```diff
@@ -250,7 +227,69 @@ Make sure to respect `YAML` identation properties, as the copied content
 must have some tabs/spaces to be correctly included under
 `gcloud-credentials.json`, the same way as the above example.
 
-## Uninstalling
+## Deploying to Your Cluster
+
+### Before Deploying
+
+Make sure you have [configured the operator](#configure-the-operator)
+correctly and have the following components installed in your machine before
+continuing:
+
+* [Make](https://www.gnu.org/software/make/). If you're running a *Unix* based
+system, it is very probable that you already have it installed. On windows,
+you may visit [this page](http://gnuwin32.sourceforge.net/packages/make.htm).
+* [Golang](https://golang.org/doc/install) to build the project.
+* [Docker](https://www.docker.com/get-started) for building and pushing the
+operator's container images.
+* [Kustomize](https://kubernetes-sigs.github.io/kustomize/installation/) to
+generate the operator's yaml files correctly.
+* [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) to
+deploy the operator on the cluster.
+  * Make sure [kubeconfig](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/)
+  is properly set up with your Kubernetes cluster data.
+* [Kubebuilder](https://github.com/kubernetes-sigs/kubebuilder#installation)
+in case you want to add controllers and other Kubernetes components.
+
```

Comment:
Many people look for quick copy/paste code that can help them get up and running quickly. It may be helpful to offer one-liners for macOS Homebrew and Ubuntu apt on how to install all these packages in one go, if packages are available for all of them.

Reply:
Awesome, thank you!